### ISSUE TITLE
DetectMultiBackend is not found

### ISSUE THREAD

**Issue by fei015** [BICLab/EMS-YOLO#3]
When I run train.py, there is a error:
"ImportError: cannot import name 'DetectMultiBackend' from 'models.common'."
In modles/common.py, class DetectMultiBackend is not found.
Looking forward to a reply！

**Comment by Fanty01** [comment]
Yeah, I also have the same problem.Looking forward to a reply too！

**Comment by L-Berry** [comment]
Yes, I have the same problem.

**Comment by wangaiping1024** [comment]
raceback (most recent call last):
  File "train.py", line 34, in <module>
    import val  # for end-of-epoch mAP
  File "/root/autodl-tmp/EMS-YOLO/val.py", line 26, in <module>   I also have the same problem. I wonder if I should add this module. My author said that I use yolov3.
    from models.common import DetectMultiBackend
ImportError: cannot import name 'DetectMultiBackend' from 'models.common' (/root/autodl-tmp/EMS-YOLO/models/common.py),

**Comment by wangaiping1024** [comment]
Looking forward to your answer from the author

**Comment by sjoks93** [comment]
Same issue here

**Comment by 666-qiong** [comment]
The common.py file misses a lot of modules

**Comment by HeuristicLU** [comment]
我跑通了，上传上来了，分享给你https://github.com/HeuristicLU/EMS-yolo

**Comment by sumingming123** [comment]
> 我跑通了，上传上来了，分享给你https://github.com/HeuristicLU/EMS-yolo

大家有遇到过这个问题吗？我已经同意了time_window设置参数，还是显示这个错误。Starting training for 250 epochs...

Epoch gpu_mem box obj cls labels img_size
0%| | 0/1 [00:00<?, ?it/s] /content/drive/MyDrive/EMS_YOLO_LU/g1_resnet/train_g1.py:343: FutureWarning: torch.cuda.amp.autocast(args...) 已弃用。请改用 torch.amp.autocast('cuda', args...)。
使用 amp.autocast(enabled=cuda):
0%| | 0/1 [00:00 <?，？it / s]
回溯（最近一次调用最后一次）：
文件“/content/drive/MyDrive/EMS_YOLO_LU/g1_resnet/train_g1.py”，第 662 行，在
main（opt）中
文件“/content/drive/MyDrive/EMS_YOLO_LU/g1_resnet/train_g1.py”，第 559 行，在主
训练（opt.hyp，opt，device，callbacks）
文件“/content/drive/MyDrive/EMS_YOLO_LU/g1_resnet/train_g1.py”，第 344 行，在训练中
pred = model（imgs）# 转发
文件“/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py”，第 1736 行，在 _wrapped_call_impl 中
返回 self._call_impl（*args，**kwargs）
文件“/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py”，第 1747 行，在 _call_impl 中
返回 forward_call(*args, **kwargs)
文件“/content/drive/MyDrive/EMS_YOLO_LU/models/yolo.py”，第 140 行，在 forward
输入[i] = x
RuntimeError: expand(torch.FloatTensor{[47, 5, 3, 320, 320]}, size=[47, 5, 3, 320]): 提供的尺寸数量 (4) 必须大于或等于张量中的维数 (5)
